### ISSUE TITLE
Unclear stance on git hooks (post-receive, etc.)

### ISSUE THREAD

**Issue by yogo1212** [go-gitea/gitea#24619]
### Description

It is unclear in which way gitea supports the in-filesystem hooks offered by git.
By the looks of it, 'Git Hooks' were once a tab in the repo settings (mentioned, for example, [here](https://discourse.gitea.io/t/mirror-gitea-repo-to-github/2085/2)).
There is an action that mentions their regeneration.

What is Gitea's position here?
Can I had hooks in the filesystem? Will they be run if they are not accessible from the menu?

### Gitea Version

HEAD

### Can you reproduce the bug on the Gitea demo site?

Yes

### How are you running Gitea?

Docker

### Database

PostgreSQL

**Comment by KN4CK3R** [comment]
Yes, but you must enable them in the config for security reasons.

**Comment by yogo1212** [comment]
@KN4CK3R I suspected something like that but I gave up after half an hour of searching through Github issues and PRs..
I missed `DISABLE_GIT_HOOKS` is in the config cheat sheet because the search also only previews webhook-related content.
https://docs.gitea.io/en-us/search/?s=hook
https://docs.gitea.io/en-us/search/?s=hooks

The existence of  a page for 'webhooks' kind of raises expectations for a similarly named 'git hooks' page.
I would've expected the docs to turn up closer information about it.

Is this worth changing the docs?
